packages/css/src/components/button/button.scss

```diff
@@ -38,42 +38,33 @@
   }
 }
 
-@mixin ams-button-forced-color-mode {
-  @media screen and (-ms-high-contrast: active), screen and (forced-colors: active) {
-    border: var(--ams-button-forced-color-mode-border); // add border because forced colors changes box-shadow to none
-  }
-}
-
 .ams-button--primary {
   background-color: var(--ams-button-primary-background-color);
-  box-shadow: var(--ams-button-primary-box-shadow);
+  border-color: var(--ams-button-primary-border-color);
+  border-style: var(--ams-button-border-style);
+  border-width: var(--ams-button-border-width);
```

Comment:
These tokens are moved to the root, the rules should be moved there as well.

Reply:
Of course.